<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ATH051663" ignored="0" name="Test" tests="4" failures="1" timestamp="23 Jul 2020 17:58:26 GMT" time="205.905" errors="0">
  <testcase name="totalValidation" time="83.95" classname="com.appium.framework.AppiumJavaFramework.ContextSwitchDemoTest"/>
  <testcase name="firstTestDemo" time="26.956" classname="com.appium.framework.AppiumJavaFramework.FirstTest"/>
  <testcase name="firstTestDemo" time="25.926" classname="com.appium.framework.AppiumJavaFramework.FirstTest"/>
  <testcase name="firstTestDemo" time="20.595" classname="com.appium.framework.AppiumJavaFramework.FailedTest">
    <failure type="java.lang.AssertionError" message="expected [Key/value is not present] but found [Preference]">
      <![CDATA[java.lang.AssertionError: expected [Key/value is not present] but found [Preference]
at com.appium.framework.AppiumJavaFramework.FailedTest.firstTestDemo(FailedTest.java:23)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- firstTestDemo -->
</testsuite> <!-- Test -->
